examples/hacker-news/client/middlewares/index.js

```diff
@@ -0,0 +1,55 @@
+import cookies from 'js-cookie'
+import {
+  USER_LOGOUT,
+  OPTIMISTIC_STORY_UPVOTED,
+  OPTIMISTIC_STORY_UNVOTED
+} from '../actions/actionTypes'
+import { rootDirectory } from '../constants'
+import { actionTypes } from 'resolve-redux'
+const { SEND_COMMAND } = actionTypes
+
+const logoutMiddleware = () => next => action => {
```

Comment:
Is `logoutMiddleware` actual since now it's performed by `resolve-auth` via special HTML form?